examples/gke-basic-tiller/main.tf

```diff
@@ -292,19 +290,20 @@ resource "null_resource" "tiller_tls_certs" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 module "tiller" {
-  source = "github.com/gruntwork-io/terraform-kubernetes-helm.git//modules/k8s-tiller?ref=v0.3.0"
+  source = "github.com/gruntwork-io/terraform-kubernetes-helm.git//modules/k8s-tiller?ref=v0.5.0"
 
-  tiller_service_account_name              = "${kubernetes_service_account.tiller.metadata.0.name}"
-  tiller_service_account_token_secret_name = "${kubernetes_service_account.tiller.default_secret_name}"
-  tiller_tls_secret_name                   = "${local.tls_secret_name}"
-  namespace                                = "${local.tiller_namespace}"
-  tiller_image_version                     = "${local.tiller_version}"
+  tiller_tls_gen_method = "none"
+  tiller_service_account_name = kubernetes_service_account.tiller.metadata[0].name
+  tiller_service_account_token_secret_name = kubernetes_service_account.tiller.default_secret_name
+  tiller_tls_secret_name = local.tls_secret_name
+  namespace = local.tiller_namespace
+  tiller_image_version = local.tiller_version
```

Comment:
This formatting always makes me sad (it comes from https://github.com/hashicorp/terraform/issues/21434)